src/app/components/SocialLinks/index.tsx

```diff
@@ -92,36 +103,79 @@ const SocialLinks = ({
   title,
   summaries = [],
   id = 'social-links-1',
+  eventTrackingData = { componentName: 'social-links' },
 }: SocialLinksProps) => {
+  const viewTracker = useViewTracker(eventTrackingData);
+  const getClickTrackerHandler = useClickTrackerHandler;
   if (!summaries.length) {
     return null;
   }
 
   const hasMultipleItems = summaries.length > 1;
 
+  const buildPromoEventTrackingData = (promo: Summary, index: number) => {
+    const itemTracker = {
+      type: 'social-link-promo',
+      text: promo.title,
+      position: index + 1,
+      resourceId: promo.id,
+    };
+    return {
+      itemTracker,
+      ...eventTrackingData,
+    };
+  };
+
   return (
     <section
       role="region"
       aria-labelledby={id}
       data-testid={id}
       css={styles.container}
+      {...viewTracker}
     >
       <Heading level={2} id={id} css={styles.heading}>
         {title}
       </Heading>
       {hasMultipleItems ? (
         <ul css={styles.unorderedList} role="list">
-          {summaries.map(summary => {
+          {summaries.map((summary, index) => {
+            const promoEventTrackingData = buildPromoEventTrackingData(
+              summary,
+              index,
+            );
+            const clickTrackerHandler = getClickTrackerHandler(
+              promoEventTrackingData,
+            );
+            const hasDescription = Boolean(summary.description);
+
             return (
               <li css={styles.item} key={summary.title}>
-                <SocialLink summary={summary} />
+                <SocialLink
+                  summary={summary}
+                  clickTrackerHandler={clickTrackerHandler}
+                />
+                {hasDescription ? (
+                  <span id={`social-link-label-${index}`}>
+                    {summary.title}
+                    <VisuallyHiddenText>{`, ${summary.description}`}</VisuallyHiddenText>
+                  </span>
+                ) : (
+                  summary.title
+                )}
               </li>
             );
           })}
         </ul>
       ) : (
         <div css={styles.item}>
-          <SocialLink summary={summaries[0]} />
+          <SocialLink
+            summary={summaries[0]}
+            eventTrackingData={eventTrackingData}
```

Comment:
Similar here, we don't need to pass `eventTrackingData` to SocialLink if it isn't being used by the component.